de.dlr.sc.virsat.swtbot.test/src/de/dlr/sc/virsat/swtbot/test/ASwtBotTestCase.java

```diff
@@ -135,7 +146,22 @@ public void tearDown() throws CoreException {
 					
 					// Now reset the workbench and remove the project
 					Activator.getDefault().getLog().log(new Status(Status.INFO, Activator.getPluginId(), "ASwtBotTestCase: Resetting Workbench"));
-					bot.resetWorkbench();
+					
+					//bot.resetWorkbench();					

```

Comment:
Is it a documented bug in SWTBot or is it something we with how we implemented the palette?

Reply:
I think it seems to be neither clearly one nor the other.  Both work like expected in isolation, but somehow interplay between the two causes trouble. Surely needs a closer look.

Reply:
OK, isee . Thus please raise an individual ticket for it, mention it here and resolve the discussion.

Reply:
Ok